besu/src/test/java/org/hyperledger/besu/cli/BesuCommandTest.java

```diff
@@ -3987,10 +3987,44 @@ public void quorumInteropEnabledSucceedsWithGasPriceSetToZero() throws IOExcepti
         "--genesis-file",
         genesisFile.toString(),
         "--min-gas-price",
-        "0");
+        "0",
+        "--privacy-public-key-file",
+        ENCLAVE_PUBLIC_KEY_PATH);
```

Comment:
what happens if privacy-public-key-file isn't specified?

Reply:
We decided (Madeline and me) that we would enable private Txs when we are compatible with GoQuorum, so we need the enclave key. In the future we might have a flag that disables private Txs ...
In short: we fail!

Reply:
And why did we need to update a test that has nothing to do privacy?